src/networks/mod.rs

```diff
@@ -172,6 +173,21 @@ pub struct ChainConfig {
     pub request_window: usize,
 }
 
+#[cfg(test)]
+impl quickcheck::Arbitrary for ChainConfig {
+    fn arbitrary(g: &mut quickcheck::Gen) -> ChainConfig {
+        use quickcheck::Arbitrary;
+        // Note: Not all `u64` values can be stored as TOML. Maybe we should use
+        // `u32` instead. Or maybe these values shouldn't be configurable at all.
+        ChainConfig {
+            network: Arbitrary::arbitrary(g),
+            genesis_cid: Arbitrary::arbitrary(g),
+            recent_state_roots: Arbitrary::arbitrary(g),
+            ..ChainConfig::default()
```

Comment:
Why?

Reply:
The config has to be serializable as TOML. But TOML does not support `u64`. Generating the `u64` fields will, correctly, cause our tests to fail. The note above talks about possible ways of fixing this.

Reply:
I think you should fix this by doing `#[arbitrary(gen(|g| u32::arbitrary(g) as u64))]`

Reply:
I'm not sure that's the best way forward. Ideally, bad states should be unrepresentable. Another option is to delete those fields entirely. I don't think we'll ever support arbitrary networks (ie, not calibnet, mainnet, or devnet).